### DISCUSSION TITLE
No puedo importar el projecto.

### DISCUSSION THREAD

**Discussion by gdrusetich** [dds-jv/foro#397] [Consultas de herramientas]
![QueMePongo](https://user-images.githubusercontent.com/81935013/233425328-7b0e8919-f405-4f8f-80d8-0d1ed3b0ae9e.png)
Buenas!
**1) Tengo esta versión:**
Apache Maven 3.9.1 (2e178502fcdbffc201671fb2537d0cb4b4cc58f8)
Maven home: C:\Program Files\apache-maven-3.9.1
Java version: 17.0.6, vendor: Oracle Corporation, runtime: C:\Program Files\Java\jdk-17
Default locale: es_AR, platform encoding: Cp1252
OS name: "windows 10", version: "10.0", arch: "amd64", family: "windows"

**2) Generé el archivo pom.xml en una carpeta QueMePongo**. Puse lo que decía en el documento de Maven, sólo cambiando el nombre del proyecto "prueba" por "QueMePongo":
<project>
  <modelVersion>4.0.0</modelVersion>
  <groupId>QueMePongo</groupId>
  <artifactId>QueMePongo</artifactId>
  <version>0.0.1-SNAPSHOT</version>
  <build>
    <plugins>
      <plugin>
        <artifactId>maven-compiler-plugin</artifactId>
        <version>3.11.0</version>
        <configuration>
          <source>17</source>
          <target>17</target>
        </configuration>
      </plugin>
    </plugins>
  </build>
</project>

3) En GitBash puse los comandos **mvn install** sobre la carpeta QueMePongo.
**Me tiró un par de Warning**.
[WARNING] Using platform encoding (Cp1252 actually) to copy filtered resources, i.e. build is platform dependent!
[WARNING] JAR will be empty - no content was marked for inclusion!

4) Abri eclipse, se me crea la carpeta .metadata en el proyecto QueMePongo.
No puedo importar el proyecto:  No projects are found to import.

¿Que puede llegar a estar pasando?
Gracias!

**Comment by flbulgarelli** [comment]
¡Hola!

¿Podrías compartirnos el proyecto a través de Github para poder verlo? O al menos, pasarnos una captura de pantalla que muestre la estructura de directorios de tu proyecto, dónde se vea la ubicación del pom.

Por otro lado, ¿cómo estás importando el proyecto en eclipse? Notá que tenés que usar _import maven project_ y no simplemente _import project_.

Respecto al warning de encoding, para solucionarlo tenés que fijar el mismo en UTF8 desde el pom: https://github.com/dds-utn/java-base-project/blob/b661f15fe1324c4f7ae03570a130be3f4fe6e9d3/pom.xml#L16 

Por último, tené en cuenta que mvn install no genera por si mismo archivos que el IDE pueda importar, sino que es el pom.xml lo que el IDE necesita para importarlo.
